### PR TITLE
Release files and update to the amplitude tracking override 📊

### PR DESCRIPTION
**Pull request by Phillip3**

### CODE REVIEW

File: App.js [hunk 1]
```diff
@@ -17,14 +18,29 @@ import AppNavigator from './navigation/AppNavigator';
 
 Amplitude.initialize(keys.AMPLITUDE);
 
-if (__DEV__) {
-  // override amplitude tracking
+const channel = Constants?.manifest?.releaseChannel;
+const emptyStorage = () =>
+  AsyncStorage.multiRemove(['@posts', '@media', '@groups', '@missions']);
+// override amplitude tracking
+if (channel === null || channel === undefined || channel === '') {
```
Comment (dereckquock) [inline App.js]: i think we can just do
```suggestion
if (!channel) {
```
here

File: App.js [hunk 2]
```diff
@@ -17,14 +18,29 @@ import AppNavigator from './navigation/AppNavigator';
...
-  // Erase all AsyncStorage
-  AsyncStorage.multiRemove(['@posts', '@media', '@groups', '@missions']);
+  emptyStorage();
+} else if (channel.indexOf('develop') !== -1) {
+  // beta testing from the store--we want to log this to amplitude, but
+  //  separate it out
+  Amplitude.logEvent = name => {
+    name = `BETA ${name}`;
+    console.log(`[amplitude]: ${name}`);
+    Amplitude.logEvent(name);
```
Comment (dereckquock) [inline App.js:36]: nice! so we'll just log the event with a `BETA` tag 🔥 
that works for me 👍 
should we just remove the `console.log`s tho?

File: android/app/src/main/assets/shell-app-manifest.json [hunk 3]
```diff
@@ -37,6 +37,10 @@
   "orientation": "portrait",
```
Comment (dereckquock) [inline android/app/src/main/assets/shell-app-manifest.json:37]: does this stuff get generated/updated when you `expo publish`?

File: App.js (hunk 2 above)
Comment (Phillip3) [reply App.js:36]: @dereckquock I kind of want it to log in the console unless we're release.

File: android/app/src/main/assets/shell-app-manifest.json (hunk 3 above)
Comment (Phillip3) [reply android/app/src/main/assets/shell-app-manifest.json:37]: It should.

Comment (dereckquock) [reply android/app/src/main/assets/shell-app-manifest.json:37]: haha i guess we'll just commit this all the time then 👍

**Review by dereckquock — APPROVED** [review]

**Pull request merged**